lib/adoptoposs/network/api/github.ex

```diff
@@ -130,6 +167,11 @@ defmodule Adoptoposs.Network.Api.Github do
     "first: #{limit}, after: \\\"#{after_cursor}\\\""
   end
 
+  defp ids_params(ids) do
```

Comment:
Purely from a perspective of interest: We might want to investigate if there is a graphql query builder if we plan to make more use of it in the future.

Reply:
I rembember that I looked into https://github.com/billperegoy/graphql_builder back then when I created the first queries.

I can't really remember why I did not go with it, though. Maybe there was something not working (yet) for these queries.
Definitely worth a look again. 👍